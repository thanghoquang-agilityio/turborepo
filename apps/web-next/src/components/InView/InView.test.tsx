import { render } from '@testing-library/react'

// Components
import { InView } from '.'

describe('InView component', () => {
  beforeEach(() => {
    const mockIntersectionObserver = jest.fn()
    mockIntersectionObserver.mockReturnValue({
      observe: () => null,
      unobserve: () => null,
      disconnect: () => null,
    })
    window.IntersectionObserver = mockIntersectionObserver
  })

  it('Should match snapshot', () => {
    const { container } = render(<InView />)
    expect(container).toMatchSnapshot()
  })
})
