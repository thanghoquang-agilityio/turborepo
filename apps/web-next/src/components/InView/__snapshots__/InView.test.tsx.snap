// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InView component Should match snapshot 1`] = `
<div>
  <div />
</div>
`;
